management/dns_update.py

```diff
@@ -303,6 +303,23 @@ def has_rec(qname, rtype, prefix=None):
 		if not has_rec(qname, rtype):
 			records.append((qname, rtype, value, explanation))
 
+	# Adds autoconfiguration A records for all domains.
+	# mta-sts.* - required A record for mta-sts (serving the policy)
+	mta_sts_records = [
+		("mta-sts", "A", env["PUBLIC_IP"], "Provides MTA-STS support"),
+		("mta-sts", "AAAA", env.get('PUBLIC_IPV6'), "Provides MTA-STS support"),
+		("_mta-sts", "TXT", "v=STSv1; id="+datetime.datetime.now().strftime("%Y%m%d%H%M%S")+"Z", "Enables MTA-STS support")
```

Comment:
While my python is not that great, it looks like the + chars would be included in the text record. In this line they wouldn't.

`("_mta-sts", "TXT", "v=STSv1;id=%sZ;" % datetime.datetime.now().strftime("%Y%m%d%H%M%S"), ".."),`

...would make this a bit more "readable"

Also the Note should add a bit more than just enables mta-sts support to help the user

Reply:
I verified, the + chars won't be included in the result, 
Your proposal worked. Thanks